apps/referral/models/referal_code.py

```diff
@@ -0,0 +1,20 @@
+# Django Imports
+from django.conf import settings
+from django.db import models
+
+
+class ReferralCode(models.Model):
+    """Referral Code model"""
+
+    code = models.CharField(max_length=56, unique=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
+        related_name='referral_codes',
+        on_delete=models.CASCADE,
+    )
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Let's add an `is_active = models.BooleanFIeld(...)` field as well.

We will need to limit the number of referral codes a user can create, otherwise they will just create infinitely many referral codes.